src/topic.ts

```diff
@@ -376,13 +370,7 @@ export class Topic {
         callback!(null, this, apiResponse!);
         return;
       }
-      let code = 0;
```

Comment:
@callmehiphop can you think of a reason why this was done that way?

Reply:
No idea to be honest, looks like it got changed when no implicit any was turned on for this file. My best guess is that maybe `err` wasn't being inferred as a `ServiceError` so this was a work around to read the code?

Reply:
Who the hell knows.  Either way, fine to delete.